README.md

```diff
@@ -35,6 +35,12 @@ This repo includes the following packages/apps:
 
 Each package/app is 100% [TypeScript](https://www.typescriptlang.org/).
 
+### How to run the complete project
```

Comment:
There's a section for this already called Development below. Let's just get rid of that and the How to run starter app section to avoid confusion